Resources/ArmTemplates/template.json

```diff
@@ -52,6 +52,13 @@
         "SharedAppServicePlanResourceGroup": {
             "type": "string",
             "defaultValue": ""
+        },
+        "CosmosDbCollectionName": {
+            "type": "string",
+            "defaultValue": "paths",
```

Comment:
I think the default value should be removed in this case.  The changes will go out to DEV & SIT at the same time so no need for a default to maintain compatibility with different environments